e2e-tests/test_simple_entity.py

```diff
@@ -2,52 +2,81 @@
 import requests
 
 
-def get_entities(backend_url, auth_header):
-    response = requests.get(f"{backend_url}/simpleEntities", headers=auth_header)
+def get_entities(backend_url, auth_header, lang):
+    if lang == "ts":
```

Comment:
As mentioned in the Python PR, let's use `simple-entities` as the route name in both backends